pkg/virtctl/ssh/wrapped.go

```diff
@@ -29,13 +27,14 @@ func (o *SSH) runLocalCommandClient(kind, namespace, name string) error {
 
 func (o *SSH) buildProxyCommandOption(kind, namespace, name string) string {
 	proxyCommand := strings.Builder{}
-	proxyCommand.WriteString("-o ProxyCommand=")
+	proxyCommand.WriteString("-o \"ProxyCommand=")
```

Comment:
Shouldnt `\"` go after `ProxyCommand=`? I.e. 
```
-o ProxyCommand="virtctl port-forward --stdio=true kind/name.namespace 22"
```
instead of
```
-o "ProxyCommand=virtctl port-forward --stdio=true kind/name.namespace 22"
```
?

Reply:
Both work for me

Reply:
I changed it as  you suggested :)